AC3.2Unit3MidunitAssessment/AC3.2Unit3MidunitAssessment/Album.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Spotify.swift
+//  AC3.2Unit3MidunitAssessment
+//
+//  Created by Ilmira Estil on 10/28/16.
+//  Copyright © 2016 C4Q. All rights reserved.
+//
+
+import Foundation
+
+internal struct Album {
+    internal let albumTitle: String
+    internal let albumArrayImages: [[String:String]]
+    internal let albumThumbnailURLString: String = ""
+    internal let albumFullImageURLString: String = ""
+    
+    
+    //String:[String:AnyObject]?
+    static func getAlbum(from data: Data) -> [Album]? {
+        do {
+            let albumJSONData: Any = try JSONSerialization.jsonObject(with: data, options: [])
+            guard let albumCasted: [String:AnyObject] = albumJSONData as? [String:AnyObject],
+                let albumArray: [AnyObject] = albumCasted["albums"] as? [AnyObject] else {return nil}
+            
+            var allAlbums = [Album]()
+            albumArray.forEach({ albumObject in
+                guard let allItems: [String:Any] = albumObject["items"] as? [String:Any],
```

Comment:
You could've made it easier on yourself by unwrapping "items" before this loop. Then looping into items, like this:

`   
 let JSONData: Any = try JSONSerialization.jsonObject(with: data, options: [])           
           
            guard let data = JSONData as? [String: Any] else { return nil }
            guard let albums = data["albums"] as? [String: Any] else { return nil }
            guard let items = albums["items"] as? [[String: Any]] else { return nil }
            
            items.forEach {
                guard let albumName = $0["name"] as? String else { return }
                guard let artistList = $0["artists"] as? [[String: Any]] else { return }
                guard let mainArtist = artistList[0]["name"] as? String else { return }
                guard let images = $0["images"] as? [[String: Any]] else { return }
                guard let thumbnail = images[2]["url"] as? String else { return }
                guard let largeImage = images[1]["url"] as? String else { return }
`